google/cloud/spanner/samples/samples.cc

```diff
@@ -1628,6 +1628,273 @@ void ListDatabaseOperationsCommand(std::vector<std::string> argv) {
   ListDatabaseOperations(std::move(client), argv[0], argv[1]);
 }
 
+//! [create-backup-schedule] [START spanner_create_backup_schedule]
```

Comment:
nit: Here and in all the new samples, the `[create-backup-schedule]` tag should be removed. It is not referenced anywhere.

Context: those are doxygen snippet tags. Some samples have these doxygen tags because handwritten clients may reference the code samples in their class documentation.

And to be explicit, the `[START ...]` tags are DevRel / cloud docs tags. We want to keep those.

Reply:
Done.